vendor/extensions/stories/app/views/refinery/stories/stories/index.html.erb

```diff
@@ -1,30 +1,42 @@
 <div class="StoriesPage">
   <div class="banner">
-    <div class="container">
-      <div class="content-column banner-content">
-        <h1>Weigh In</h1>
+    <div class="container"></div>
+  <div class="StoriesPage">
+    <div class="weigh-in__header">
+      <div class="weigh-in__header--content">
+        <div class="weigh-in__header--title">Weigh in...</div>
+        <p class="weigh-in__header--text">
+          Opinions, ideas, and priorities that spring from the lived experience of the people of the region are an 
+          important part of MetroCommon. We’re collecting public input at every stage of the planning process.
+        <br />
+        <br />
```

Comment:
Instead of using `<br />` here would you consider surrounding each paragraph in a `<p>` tag?